code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2092,3 +2092,21 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_icon_state = "planet_cracker"
 	glass_name = "Planet Cracker"
 	glass_desc = "Although historians believe the drink was originally created to commemorate the end of an important conflict in man's past, its origins have largely been forgotten and it is today seen more as a general symbol of human supremacy."
+
+/datum/reagent/consumable/ethanol/mauna_loa
+	name = "Mauna Loa"
+	description = "Extremely hot; not for the faint of heart!"
+	boozepwr = 40
+	color = "#fe8308" // 254, 131, 8
+	quality = DRINK_FANTASTIC
+	taste_description = "firey, with an aftertaste of burnt flesh"
```

Comment:
Generally, taste_descriptions are nouns, not adjectives ("with a hint of firey, with an aftertaste of burnt flesh" sounds weird, no?). Also, you spelled "fiery" wrong. I'd recommend changing this to "fire, with an aftertaste of burnt flesh".

EDIT: Oh, hm, it looks like some other drinks also use adjectives in their descriptions. Go with whatever you want, but I do recommend correcting your spelling of "fiery".

Reply:
Fixed the typo... One of many. I'll try to do better if I submit anything more in the future, I've had a lot of these.